src/frontend/components/TeacherLiveStopConfirmation/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { Box, Button, Heading, Paragraph, Spinner } from 'grommet';
+import React, { Fragment, useEffect, useState } from 'react';
+import toast from 'react-hot-toast';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { endLive } from 'data/sideEffects/endLive';
+import { useStopLiveConfirmation } from 'data/stores/useStopLiveConfirmation';
+import { useVideo } from 'data/stores/useVideo';
+import { liveState, Video } from 'types/tracks';
+
+const messages = defineMessages({
+  alertInfo: {
+    defaultMessage:
+      'You are about to stop the live.{br}Every viewers will be disconnected.{br}Are you sure you want to stop this Live ?',
```

Comment:
I feel like this message will be split in the future.

Reply:
WDYT @lunika ?

Reply:
split where ?

Reply:
At each `{br}`

Reply:
@lunika @kernicPanel what about that ?

Reply:
If needed we will split it in the future.